nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/trash/AbstractTrashService.java

```diff
@@ -320,10 +324,10 @@ public String unmangleName(DocumentModel doc) {
                 if (session != null) {
                     String orig = matcher.group(1);
                     String parentPath = session.getDocument(doc.getParentRef()).getPathAsString();
-                    if (parentPath.equals("/")) {
+                    if (parentPath.equals(PATH_SEPARATOR)) {
```

Comment:
its always better to compare constant value to the var `PATH_SEPARATOR. equals(parentPath)`